src/index.html

```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Colored blocks</h1>
+  <div class  = "first_rect"></div>
```

Comment:
```suggestion
  <div class="first_rect"></div>
```
пробелы не нужны, исправь везде